user/start.S

```diff
@@ -0,0 +1,33 @@
+#include <mips/asm.h>
+
+    .set noreorder
+    .global _start
+
+# This is the starting procedude for user ELF images, used as their
+# entry point. It takes care of preparing argc and argc for main().
+# For details about environment in which this procedure starts, refer
+# to prepare_program_stack in exec.c.
+
+LEAF(_start)
```

Comment:
To be honest, I don't really understand these, and I don't know how to choose them correctly.